src/main/java/net/logstash/logback/composite/FormattedTimestampJsonProvider.java

```diff
@@ -160,11 +96,11 @@ protected String getFormattedTimestamp(Event event) {
      */
     private void updateTimestampWriter() {
         if (UNIX_TIMESTAMP_AS_NUMBER.equals(pattern)) {
-            timestampWriter = new NumberTimestampWriter();
+            timestampWriter = null;
         } else if (UNIX_TIMESTAMP_AS_STRING.equals(pattern)) {
-            timestampWriter = new StringTimestampWriter();
+            timestampWriter = DateTimeFormatter.ofPattern(DEFAULT_PATTERN).withZone(TimeZone.getDefault().toZoneId());
```

Comment:
This is a backwards incompatible change that I won't be able to accept.

logstash-logback-encoder still needs to optionally support writing timestamps as strings indicating the number of milliseconds since unix epoch

Reply:
I missunderstood the StringTimestampWriter, I will take a second look